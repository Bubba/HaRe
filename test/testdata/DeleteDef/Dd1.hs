module DeleteDef.Dd1 where

--simple bind
c :: Integer
c = 5

remove5s :: [Integer] -> [Integer]
remove5s [] = []
remove5s (5:xs) = remove5s xs
remove5s (x:xs) =  x: (remove5s xs)
