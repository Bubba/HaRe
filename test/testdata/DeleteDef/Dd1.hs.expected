module DeleteDef.Dd1 where



remove5s :: [Integer] -> [Integer]
remove5s [] = []
remove5s (5:xs) = remove5s xs
remove5s (x:xs) =  x: (remove5s xs)
